src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayload.cs

```diff
@@ -92,51 +98,52 @@ public bool TryGetBlock(out Block? block, UInt256? totalDifficulty = null)
             }
         }
 
-        public Keccak ParentHash { get; set; } = null!;
-        public Address FeeRecipient { get; set; }
-        public Keccak StateRoot { get; set; } = null!;
-        public Keccak ReceiptsRoot { get; set; } = null!;
+        public Keccak ParentHash { get; set; } = Keccak.Zero;
+        public Address FeeRecipient { get; set; } = Address.Zero;
+        public Keccak StateRoot { get; set; } = Keccak.Zero;
+        public Keccak ReceiptsRoot { get; set; } = Keccak.Zero;
 
-        [JsonProperty(NullValueHandling = NullValueHandling.Include)]
+        //[JsonProperty(NullValueHandling = NullValueHandling.Include)]
         public Bloom LogsBloom { get; set; } = Bloom.Empty;
         public Keccak PrevRandao { get; set; } = Keccak.Zero;
 
-        [JsonProperty(NullValueHandling = NullValueHandling.Include)]
+        //[JsonProperty(NullValueHandling = NullValueHandling.Include)]
         public long BlockNumber { get; set; }
         public long GasLimit { get; set; }
         public long GasUsed { get; set; }
         public ulong Timestamp { get; set; }
         public byte[] ExtraData { get; set; } = Array.Empty<byte>();
         public UInt256 BaseFeePerGas { get; set; }
 
-        [JsonProperty(NullValueHandling = NullValueHandling.Include)]
-        public Keccak? BlockHash { get; set; } = null!;
+        //[JsonProperty(NullValueHandling = NullValueHandling.Include)]
+        public Keccak BlockHash { get; set; } = Keccak.Zero;
 
         /// <summary>
         /// Array of transaction objects, each object is a byte list (DATA) representing TransactionType || TransactionPayload or LegacyTransaction as defined in EIP-2718
         /// </summary>
         public byte[][] Transactions { get; set; } = Array.Empty<byte[]>();
 
-        public override string ToString() => BlockHash == null ? $"{BlockNumber} null" : $"{BlockNumber} ({BlockHash})";
+        /// <summary>
+        /// Gets or sets an RLP-encoded collection of <see cref="Withdrawal"/> as defined in
+        /// <see href="https://eips.ethereum.org/EIPS/eip-4895">EIP-4895</see>.
+        /// </summary>
+        public IEnumerable<byte[]>? Withdrawals { get; set; }
```

Comment:
From spec:
1. If withdrawal functionality is activated, client software **MUST** return an `INVALID` status with the appropriate `latestValidHash` if `payload.withdrawals` is `null`.

and we're not passing this condition